apps/backend/file-service/src/test/resources/application-test.yml

```
@@ -8,14 +8,15 @@ spring:
-      schema-locations:
-        - classpath:schema.sql
-        - classpath:functions/*.sql
\ No newline at end of file
+      ddl-auto: none
+  flyway:
+    enabled: true
+    locations: classpath:db/migration
+    baseline-on-migrate: true
+    validate-on-migrate: true
+    table: file_service_flyway_schema_history
+    baseline-version: 0
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)